### PR TITLE
add few words

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Review by karl-johan-grahn — APPROVED** [review]

File: styles/Vocab/Stakater/accept.txt [hunk 1]
```diff
@@ -242,3 +242,5 @@ xfs
 yaml
 YAML
 Zerotier
+NDA[s]?
+Signup
```
Comment (karl-johan-grahn) [inline styles/Vocab/Stakater/accept.txt:246]: signup can be lowercase too

**Review by karl-johan-grahn — CHANGES_REQUESTED** [review]

File: styles/Vocab/Stakater/accept.txt (hunk 1 above)
Comment (karl-johan-grahn) [reply styles/Vocab/Stakater/accept.txt:246]: you can also just write "sign up" which is accepted, so no need to add this

Comment (rasheedamir) [reply styles/Vocab/Stakater/accept.txt:246]: On websites we see Signup don't we?

Comment (karl-johan-grahn) [reply styles/Vocab/Stakater/accept.txt:246]: which website?

**Review by karl-johan-grahn — APPROVED** [review]

**Pull request merged**